src/delegate.rs

```diff
@@ -87,6 +87,9 @@ pub trait LanguageServerCore {
     #[rpc(name = "textDocument/completion", raw_params)]
     fn completion(&self, params: Params) -> BoxFuture<Option<CompletionResponse>>;
 
+    #[rpc(name = "completionItem/resolve", raw_params)]
+    fn resolve(&self, params: Params) -> BoxFuture<CompletionItem>;
```

Comment:
I just realized this is a bad idea, because it clashes with `code_lens_resolve`, I'll change it to `completion_resolve`.

Reply:
No worries! I was just about to suggest the same thing. Thanks for preempting me in that regard, haha.